adafruit_ble_radio.py

```diff
@@ -142,19 +142,19 @@ def send_bytes(self, message):
         time.sleep(AD_DURATION)
         self.ble.stop_advertising()
 
-    def receive(self):
+    def receive(self, **args):
         """
         Returns a message received on the channel on which the radio is
         listening.
 
         :return: A string representation of the received message, or else None.
         """
-        msg = self.receive_full()
+        msg = self.receive_full(**args)
```

Comment:
If we used the named argument for the receive function as mentioned above this could be `wait=wait` or whatever the name of the argument ends up being instead of `**args`

Reply:
I changed wait to timeout and used `timeout=timeout` as suggested (though it took a second commit to get it right, haha).